CONTRIBUTING.md

```diff
@@ -30,24 +41,37 @@ mamba activate xdem-dev  # Or any other name specified above
 ### Tests
 
 At least one test per feature (in the associated `tests/test_*.py` file) should be included in the PR, using `pytest` (see existing tests for examples).
+The structure of test modules and functions in `tests/` largely mirrors that of the package modules and functions in `xdem/`.
 
 To run the entire test suite, run `pytest` in the current directory:
 ```bash
 pytest
 ```
 
+Running `pytest` will trigger a script that automatically downloads test data from [https://github.com/GlacioHack/xdem-data](https://github.com/GlacioHack/xdem-data) before running all tests.
+
+### Documentation
+
+If your changes need to be reflected in the documentation, update the related pages located in `doc/source/`. The documentation is written in MyST markdown syntax, similar to GitHub's default Markdown (see [MyST-NB](https://myst-nb.readthedocs.io/en/latest/authoring/text-notebooks.html) for details).
+
+To ensure that the documentation is building properly after your changes, you can run `pytest tests/test_doc.py` on Linux. On Windows and Mac, the documentation is not maintained, so you can wait to open the PR for it to be checked by the CI.
```

Comment:
I believe there is a command to build the documentation locally, no? It would be great to add it here. Everytime I make changes to the documentation, I forget about it...

Reply:
Added!

Reply:
(but FYI: `tests/test_doc` also builds the documentation, so does exactly the same thing more easily)